scene/2d/collision_object_2d.cpp

```diff
@@ -78,6 +78,10 @@ void CollisionObject2D::_notification(int p_what) {
 
 		case NOTIFICATION_TRANSFORM_CHANGED: {
 			if (only_update_transform_changes) {
+				Node2D *synchronized_body_parent = Object::cast_to<Node2D>(get_parent());
+				if (synchronized_body_parent && synchronized_body_parent->is_class("PathFollow2D")) {
```

Comment:
Any reason for using the "is_class" instead of casting to the type directly?

Reply:
I was getting something like C2065 undeclared identifier for the PathFollow2D. I was checking the line GDCLASS(PathFollow2D, Node2D); from path_2d.h. Isn't that enough to make PathFollow2D global?

Reply:
You need to include the header for that to be available, this isn't GDScript you don't have global classes